src/amo/sagas/addonsByAuthors.js

```diff
@@ -0,0 +1,60 @@
+// Disabled because of
+// https://github.com/benmosher/eslint-plugin-import/issues/793
+/* eslint-disable import/order */
+import { all, call, put, select, takeEvery } from 'redux-saga/effects';
+/* eslint-enable import/order */
+
+import {
+  FETCH_OTHER_ADDONS_BY_AUTHORS,
+  loadOtherAddonsByAuthors,
+} from 'amo/reducers/addonsByAuthors';
+import searchApi from 'core/api/search';
+import log from 'core/logger';
+import { createErrorHandler, getState } from 'core/sagas/utils';
+
+
+const NB_ADDONS_MAX_PER_AUTHOR = 6;
+
+export function* fetchOtherAddonsByAuthors({ payload }) {
+  const { errorHandlerId, authors, slug } = payload;
+  const errorHandler = createErrorHandler(errorHandlerId);
+
+  yield put(errorHandler.createClearingAction());
+
+  try {
+    const state = yield select(getState);
+
+    const responses = yield all(authors.map((author) => call(searchApi, {
+      api: state.api,
+      filters: { author, page_size: NB_ADDONS_MAX_PER_AUTHOR },
```

Comment:
This should probably [sort by hotness](https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#addon-search-sort) or something

Reply:
Of course yes!